components/gitpod-protocol/src/protocol.ts

```diff
@@ -154,6 +157,12 @@ export interface AdditionalUserData {
     dotfileRepo?: string;
     // Identifies an explicit team or user ID to which all the user's workspace usage should be attributed to (e.g. for billing purposes)
     usageAttributionId?: string;
+    // when was the last time the user updated their profile information or has been nudged to do so.
+    lastUpdatedDetailsNudge?: string;
+    // the user's company name
+    companyName?: string;
```

Comment:
idea: Would it make sense to group these two attributes (companyName and emailAddress) under e.g. `profile`? The motivation would be clarity: `additionalUserData` is already pretty crowded, and is degenerating into a "kitchen sink" rather quickly...